Framework/DataHandling/inc/MantidDataHandling/DefaultEventLoader.h

```diff
@@ -88,7 +92,7 @@ template <class T> void DefaultEventLoader::makeMapToEventLists(std::vector<std:
   vectors.resize(m_ws.nPeriods());
   if (event_id_is_spec) {
     // Find max spectrum no
-    auto *ax1 = m_ws.getAxis(1);
+    const auto *ax1 = m_ws.getAxis(1);
     specnum_t maxSpecNo = -std::numeric_limits<specnum_t>::max(); // So that any number will be
```

Comment:
Nitpicking: isn't the minimum value actually `-std::numeric_limits<specnum_t>::max() - 1`?  :)

Reply:
`specnum_t = int32_t` so I'm not sure you can go one below or not. In practice, the minimum spectrum number is more like zero.